test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -165,6 +165,7 @@ public void GetTableDefinitionWithSchemaTest()
             connectionResult.TextDocumentPosition = textDocument;
             var languageService = LanguageService.Instance;
             await languageService.UpdateLanguageServiceOnConnection(connectionResult.ConnectionInfo);
+            Thread.Sleep(2000);
```

Comment:
The test is failing because of the shared LanguageService instance.  This is causing an exception in the Blocking Collection when it runs alongside a test that also uses the same instance. I'm refactoring the test to avoid this. Closing the PR until I'm done refactoring.